src/components/game/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+
+import story from './game.js'
+
+interface IGameProps {
+  canvas?: HTMLCanvasElement
+  handleSaveCanvas: (c: HTMLCanvasElement) => void
+}
+
+export type DispatchProps = Pick<IGameProps, 'handleSaveCanvas'>
+
+export type StateProps = Pick<IGameProps, 'canvas'>
+
+export class Game extends React.Component<IGameProps, {}> {
+  private canvas: HTMLCanvasElement
+  private div: HTMLDivElement
+
+  public componentDidMount() {
+    /**
+     * Basically, if the function story is called twice (on different canvas
+     * elements), the second time the component is mounted, the pixi.js canvas
+     * will show nothing but a black screen. This means that if the user
+     * navigate aways from the game tab, and then back again, the game would not
+     * work.
+     *
+     * So, we save a reference to the first canvas that is loaded. Thereafter,
+     * when this component is mounted, use that canvas instead of the new canvas
+     * mounted with this div. This is a bit hacky, and refs aren't favoured in
+     * react, but it also prevents excessive loading of the game
+     */
+    if (this.props.canvas === undefined) {
```

Comment:
@evansb I think the loading of the game takes up quite a bit of time, and this would be a good way to maintain continuity in the SPA. Is there any better way/optimisation that you might be able to advise us on?